pkg/apis/clusterlink.net/v1alpha1/instance_types.go

```diff
@@ -50,8 +50,8 @@ const (
 )
 
 const (
-	// ExternalPort represents the default value for the external ingress service of the LoadBalancer type.
-	ExternalPort = 443
+	// ExternalDefaultPort represents the default value for the external ingress service of the LoadBalancer type.
+	ExternalDefaultPort = 443
```

Comment:
```suggestion
	DefaultLoadBalancerPort = 443
```
Prefix default values with `Default` and indicate this is `LoadBalancer` specific.
Obviously needs to change usages as well.